lib/Common/Exceptions/Throw.cpp

```diff
@@ -82,6 +82,14 @@ namespace Js {
         RaiseException((DWORD)DBG_TERMINATE_PROCESS, EXCEPTION_NONCONTINUABLE, 0, NULL);
     }
 
+#ifdef ENABLE_JS_BUILTINS
+    void Throw::FatalJsBuiltInError()
+    {
+        AssertMsg(false, "Could not initialize JsBuiltIns!");
+        ReportFatalException(NULL, E_UNEXPECTED, Fatal_JsReentrancy_Error, 0);
```

Comment:
Should this have its own error reason instead of just `Fatal_JsReentrancy_Error`?

Reply:
Yes! Fixed